catalog/message/catalog.json

```diff
@@ -25,31 +26,27 @@
       "dcat:keyword": [
```

Comment:
The dct:description and dcat:keyword should be a Multilanguage array instead of a plain string or array of plain strings.

Reply:
Let's have a discussion about it. I am open for both.